### PR TITLE
Docs: Added basic GUI for rollout deployment

### PR DESCRIPTION
**Pull request by SNe789**
# Description

- Deployment Template section is modified.
- Added Basic GUI Configuration.

### CODE REVIEW

**Comment by prakarsh-dt** [comment]
Documentation for #2488

File: docs/user-guide/creating-application/deployment-template.md [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
-* On the **Deployment Template** page, select the drop-down under **Chart type**.
+* On the **Base Deployment Template** page, select the drop-down under **Chart type**.
 
 You can select a chart in one of the following ways:
 
-## Select chart by Devtron
-
 1. [Rollout Deployment](deployment-template/rollout-deployment.md)
-2. [Job and Cronjob](deployment-template/job-and-cronjob.md)
+2. [Cronjob & Job](deployment-template/job-and-cronjob.md)
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/deployment-template.md:17]: Is it Cronjob & Job on the UI?

Comment (SNe789) [reply docs/user-guide/creating-application/deployment-template.md:17]: It is existing as Cronjob & Job in UI, so I changed the same in doc also (but later I realised what if the UI is wrong or to be updated), so I reverted the changes. This has to be confirmed. Moreover, there is a new feature where a new deployment helm chart is to be added. So, we can have a look more in detail on all the charts including Cronjob and Job.

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 2]
```diff
@@ -64,7 +65,7 @@ Here we can pass the list of env variables , every record is an object which con
 
 To set environment variables for the containers that run in the Pod.
 
-### Example of env variables
+### Example of Envvariables
```
[the remark this answers is not in the document]
Comment (SNe789) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:68]: Yes, its a mistake. I thought I changed as per YAML. I will update it.

[the remark this answers is not in the document]
Comment (SNe789) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md]: updated.

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**